setup.cfg

```diff
@@ -3,6 +3,8 @@
 pep8maxlinelength = 120
 pep8ignore =
     docs/conf.py ALL
+    *.py W503
```

Comment:
Please add a short comment what these rules are that we ignore

Reply:
done. Also, E126 and W503 are commonly ignored in standard rule sets, since people have different views on them.